pages/_app.js

```diff
@@ -68,29 +69,7 @@ export default class MyApp extends App {
 
   render () {
     return this.state.hasError ? (
-      <section>
-        <h1>There was an error!</h1>
-        <p>
-          <a
-            href='#'
-            onClick={() =>
-              Sentry.showReportDialog({ eventId: this.state.errorEventId })
-            }
-          >
-            📣 Report this error
-          </a>
-        </p>
-        <p>
-          <a
-            href='#'
-            onClick={() => {
-              window.location.reload(true)
-            }}
-          >
-            Or, try reloading the page
-          </a>
-        </p>
-      </section>
+      <ErrorPage errorCode={500} />
```

Comment:
Maybe what you can do is pass in here the `errorEventId` and the `errorInfo` to the `ErrorPage`?